model/project.py

```diff
@@ -0,0 +1,200 @@
+"""Represent a team project."""
+import hashlib
+import time
+
+
+class Project:
+    """Represent a team project with team ID and related fields and methods."""
+
+    def __init__(self, github_team_id, github_urls):
+        """
+        Initialize the team project.
+
+        Project ID is just the current epoch time and the first github project
+        URL mashed together using SHA1.
```

Comment:
Can't you just say `project id is sha1 generated from github project url and epoch time`?

Reply:
ok